ui/src/containers/CreateVolume.js

```diff
@@ -420,522 +296,580 @@ const CreateVolume = (props) => {
-                            </Tooltip>
-                          </>
-                        }
-                        error={touched.path && errors.path}
-                        onBlur={handleOnBlur}
+                    }
+                  />
+
+                  <FormGroup
+                    label={intl.formatMessage({ id: 'force_lvcreate' })}
+                    helpErrorPosition="bottom"
+                    labelHelpTooltip={intl.formatMessage({
+                      id: 'force_lvcreate_explanation',
+                    })}
+                    error={touched.forceLVCreate && errors.forceLVCreate}
+                    id="forceLVCreate"
+                    content={
+                      <Toggle
+                        id="forceLVCreate"
+                        name="forceLVCreate"
+                        toggle={values.forceLVCreate}
+                        label={values.forceLVCreate ? 'Yes' : 'No'}
+                        onChange={() => {
+                          setFieldValue('forceLVCreate', !values.forceLVCreate);
+                        }}
                       />
-                    );
-                  case LVM_LOGICAL_VOLUME:
-                    return (
-                      <>
-                        {sizeInputCombinedField()}
-                        <Input
-                          name="vgName"
-                          value={values.vgName}
-                          onChange={handleChange('vgName')}
-                          label={`Volume Group Name*`}
-                          error={touched.vgName && errors.vgName}
-                          onBlur={handleOnBlur}
-                        />
-                      </>
-                    );
-                  default:
-                    return;
+                    }
+                  />
+                </>
+              );
+            default:
+              return;
+          }
+        };
+
+        return (
+          <Form
+            onSubmit={handleSubmit}
+            rightActions={
+              <Stack>
+                <Button
+                  label={intl.formatMessage({ id: 'cancel' })}
+                  type="button"
+                  variant="outline"
+                  onClick={() => history.goBack()}
+                />
+                <Button
+                  label={intl.formatMessage({ id: 'create' })}
+                  type="submit"
+                  variant={'primary'}
+                  disabled={!dirty || !isEmpty(errors) || values.labelName}
+                  data-cy="submit-create-volume"
+                />
+              </Stack>
+            }
+            layout={{
+              kind: 'page',
+              title: intl.formatMessage({ id: 'create_volume' }),
+            }}
+            banner={
+              isStorageClassExist ? null : (
+                <Banner
+                  variant="warning"
+                  icon={<i className="fas fa-exclamation-triangle" />}
```

Comment:
Let's use the <Icon/> from core-ui ;) 

```suggestion
                  icon={<Icon name="Exclamation-triangle" />}
```